packages/lambdas/src/sqs-to-converter.ts

```diff
@@ -46,6 +46,7 @@ const ossApi = apiClient(apiURL);
 function replaceIDs(fhirBundle: FHIRBundle, patientId: string): FHIRBundle {
   const stringsToReplace: { old: string; new: string }[] = [];
   for (const bundleEntry of fhirBundle.entry) {
+    if (!bundleEntry.resource.id) throw new Error(`Missing resource id`);
```

Comment:
we'll probably want to update the invocation of `replaceIDs()` and wrap it in a try catch, so we can capture the extras on sentry - ie the file name that errored in specific so we can easily find on s3

unless that'll already be captured by default?

Reply:
discussed offline, apparently the sentry wrapper will capture the info we need